crypto/context.c

```diff
@@ -583,6 +588,8 @@ void *ossl_lib_ctx_get_data(OSSL_LIB_CTX *ctx, int index)
         return ctx->store_loader_store;
     case OSSL_LIB_CTX_SELF_TEST_CB_INDEX:
         return ctx->self_test_cb;
+    case OSSL_LIB_CTX_RCU_LOCAL_KEY_INDEX:
+        return &ctx->rcu_local_key;
```

Comment:
Is this intentionally in the FIPS_MODULE ifndef? Also I wonder if we really want to use ossl_lib_ctx_get_data() for any further libctx members. IMO it is just an artefact of the original implementation of these data in 3.0 version but anything newly added could be accessed via strongly typed accessor.

Reply:
no, thats an error.  it should be outside of that.  I can add an accessor as well